packages/manager/modules/pci/src/components/hds/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import component from './component';
+
+const moduleName = 'hdsOption';
```

Comment:
```suggestion
const moduleName = 'pciProjectHdsOption';
```

Reply:
Thanks.
I renamed it `ovhManagerPciComponentsProjectHdsOption` like used pattern for others PCI component.